elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -682,10 +681,31 @@ class PfeNavigation extends PFElement {
 
       // Convert dropdown links into buttons
       const dropdownButton = document.createElement("button");
+      // Select mobile menu toggle button
+      const mobileMenuToggleButton = this.shadowRoot.getElementById("mobile__button");
```

Comment:
Some of these elements have pointers set on the object instance, you can use that instead of re-selecting the element:
https://github.com/patternfly/patternfly-elements/blob/CPFED-3689-new-navigation-epic/elements/pfe-navigation/src/pfe-navigation.js#L91-L104

Reply:
@wesruv I updated to using the pointers already created and am setting the initial aria settings using an updated version of this._addCloseDropdownAttributes(). I updated that function to also set the aria-controls to the id of the corresponding content panels.